_config.yml

```diff
@@ -18,6 +18,27 @@ facebook_username:  pclubiitk
 disqus_id: 
 
 excoordies:
+  '2018 - 19':
+    'Ahsan Barkati':
+      mail: ahsanb@iitk.ac.in
+      github: ahsanbarkati
+      image: ahsan.jpg
+      body:
+    'Anant Prakash':
+      mail: anantp@iitk.ac.in
+      github: apwhitehat
+      image: anant.jpg
+      body:
+    'Ayush Bansal':
+      mail: ayushb@iitk.ac.in
+      github: ayush268
+      image: ayush.jpg
+      body:
+    'Parv Mor':
```

Comment:
Strictly speaking, Parv was relieved of his responsibilities. We may want to remove him from this list.

Reply:
Done